src/AIAD.Api/Startup.cs

```diff
@@ -45,8 +39,8 @@ public void ConfigureServices(IServiceCollection services)
             });
 
             #region JWT
-            var jwtAppSettings = this.Configuration.GetSection("Jwt").Get<JwtAppSettings>();
-            services.AddSingleton<JwtAppSettings>(jwtAppSettings);
+            services.Configure<JwtAppSettings>(Configuration.GetSection("Jwt"));
+            var jwtAppSettings = services.BuildServiceProvider().GetRequiredService<JwtAppSettings>();
```

Comment:
This throws a runtime exception because `JwtAppSettings` isn't registered.

Reply:
Resolved